lexos/static/js/scripts_cut.js

```diff
@@ -310,7 +310,14 @@ function jsonifyForm () {
 function downloadCutting () {
   // Unfortunately, you can't trigger a download with an ajax request; calling a
   // Flask route seems to be the easiest method.
-  window.location = '/downloadCutting'
+    if ($('#num_active_files').val() > '0') {
+        window.location = '/downloadCutting'
+    }
+    else {
+        $('#error-modal').modal()
```

Comment:
just modal, without error message?

Reply:
This actually worked fine before anyway with the correct message and proper links to upload and manage but it is a little weird since I wasn't defining the error message so i fixed it.